test/Microsoft.Azure.Devices.Edge.Test/helpers/ManualProvisioningFixture.cs

```diff
@@ -19,12 +19,14 @@ public class ManualProvisioningFixture : BaseFixture
         protected readonly IotHub iotHub;
         protected IEdgeDaemon daemon;
 
+        public ManualProvisioningFixture(string connectionString, string eventHubEndpoint)
```

Comment:
These two constructors can be one constructor like this:
```csharp
public ManualProvisioningFixture(
    string connectionString = Context.Current.ConnectionString,
    string eventHubEndpoint = Context.Current.EventHubEndpoint)
{
    this.iotHub = new IotHub(connectionString, eventHubEndpoint, Context.Current.Proxy);
}
```

Reply:
Same problem as above - Using that method, the compiler complains because Context.Current.* aren't compile-time constant